UndertaleModTool/Scripts/Community Scripts/TouchControlsEnabler.csx

```diff
@@ -0,0 +1,452 @@
+{
+    Name = new UndertaleString("PageItem " + ++lastTextPageItem),
+    SourceX = 61, SourceY = 83, SourceWidth = 79, SourceHeight = 9,
+    TargetX = 0, TargetY = 0, TargetWidth = 79, TargetHeight = 9,
+    BoundingWidth = 79, BoundingHeight = 9,
+    TexturePage = textures["controls.png"]
+};
+Data.TexturePageItems.Add(pg_button_scale);
+
+UndertaleTexturePageItem pg_analog_scale = new UndertaleTexturePageItem()
+{
+    Name = new UndertaleString("PageItem " + ++lastTextPageItem),
+    SourceX = 61, SourceY = 94, SourceWidth = 79, SourceHeight = 12,
+    TargetX = 0, TargetY = 0, TargetWidth = 79, TargetHeight = 12,
+    BoundingWidth = 79, BoundingHeight = 12,
+    TexturePage = textures["controls.png"]
+};
+Data.TexturePageItems.Add(pg_analog_scale);
+
+UndertaleTexturePageItem pg_analog_type = new UndertaleTexturePageItem()
+{
+    Name = new UndertaleString("PageItem " + ++lastTextPageItem),
+    SourceX = 61, SourceY = 108, SourceWidth = 72, SourceHeight = 12,
+    TargetX = 0, TargetY = 0, TargetWidth = 72, TargetHeight = 12,
+    BoundingWidth = 72, BoundingHeight = 12,
+    TexturePage = textures["controls.png"]
+};
+Data.TexturePageItems.Add(pg_analog_type);
+
+UndertaleTexturePageItem pg_reset_config = new UndertaleTexturePageItem()
+{
+    Name = new UndertaleString("PageItem " + ++lastTextPageItem),
+    SourceX = 61, SourceY = 122, SourceWidth = 79, SourceHeight = 12,
+    TargetX = 0, TargetY = 0, TargetWidth = 79, TargetHeight = 12,
+    BoundingWidth = 79, BoundingHeight = 12,
+    TexturePage = textures["controls.png"]
+};
+Data.TexturePageItems.Add(pg_reset_config);
+
+UndertaleTexturePageItem pg_controls_opacity = new UndertaleTexturePageItem()
+{
+    Name = new UndertaleString("PageItem " + ++lastTextPageItem),
+    SourceX = 61, SourceY = 136, SourceWidth = 107, SourceHeight = 13,
+    TargetX = 0, TargetY = 0, TargetWidth = 107, TargetHeight = 13,
+    BoundingWidth = 107, BoundingHeight = 13,
+    TexturePage = textures["controls.png"]
+};
+Data.TexturePageItems.Add(pg_controls_opacity);
+
+UndertaleTexturePageItem pg_arrow_leftright = new UndertaleTexturePageItem()
+{
+    Name = new UndertaleString("PageItem " + ++lastTextPageItem),
+    SourceX = 142, SourceY = 83, SourceWidth = 41, SourceHeight = 9,
+    TargetX = 0, TargetY = 0, TargetWidth = 41, TargetHeight = 9,
+    BoundingWidth = 41, BoundingHeight = 9,
+    TexturePage = textures["controls.png"]
+};
+Data.TexturePageItems.Add(pg_arrow_leftright);
+
+UndertaleTexturePageItem pg_black = new UndertaleTexturePageItem()
+{
+    Name = new UndertaleString("PageItem " + ++lastTextPageItem),
+    SourceX = 0, SourceY = 0, SourceWidth = 640, SourceHeight = 480,
+    TargetX = 0, TargetY = 0, TargetWidth = 640, TargetHeight = 480,
+    BoundingWidth = 640, BoundingHeight = 480,
+    TexturePage = textures["black.png"]
+};
+Data.TexturePageItems.Add(pg_black);
+
+var spr_joybase = new UndertaleSprite()
```

Comment:
You should also be able to write a convenience function for these. 
Can probably make the signature be `string name, int width, int height, TexturePageItem[] sprites`

Reply:
oh wait I updated the script but didn't pay attention to `TexturePageItem[] sprites`, I should've passed textures and made it inside the function to, will do a bit later.

Reply:
done.